apps/studio/src/features/editing-experience/atoms.ts

```diff
@@ -3,3 +3,12 @@ import { atom } from "jotai"
 import type { PendingMoveResource } from "./types"
 
 export const moveResourceAtom = atom<null | PendingMoveResource>(null)
+
+export interface CollectionLinkProps {
+  ref: string
+  summary?: string
```

Comment:
issue: same comment as the one in the schema, should be called `description` so that it can be read by the parent collection page.

Reply:
oh yes, updated